### DISCUSSION TITLE
【灌水】还是X浏览器登录绳网快

### DISCUSSION THREAD

**Discussion by FFT66** [share121/inter-knot#2208] [其他]
很快

**Comment by shydrogen** [comment]
不如咕噜咕噜（Google）<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by misaka-lastorder** [comment]
建议用一个叫fairy的ai
